packages/backend/src/utils.ts

```diff
@@ -0,0 +1,23 @@
+import commander from 'commander'
+
+interface Options {
+  platform?: any
+  dataPath?: any
+  dataPort?: any
+  torBinary?: any
+  authCookie?: any
+  controlPort?: any
+  httpTunnelPort?: any
+  appDataPath?: string
+  socketIOPort?: number
+  resourcesPath?: string
+  socketIOSecret: string
+}
+
+// concept
+export const validateOptions = (_options: commander.OptionValues) => {
+  const options = _options as Options
+  if (!options.socketIOSecret) {
+    throw new Error('socketIOSecret is missing in options')
+  }
+}
```

Comment:
I would probably call this file config.ts or options.ts, might make it easier to find